client/me/purchases/vat-info/index.tsx

```diff
@@ -84,11 +100,57 @@ export default function VatInfoPage() {
 					</CardHeading>
 					<p className="vat-info__sidebar-paragraph">
 						{ translate(
-							/* translators: %s is the name of taxes in the country (eg: "VAT" or "GST"). */
-							"We currently only provide %(taxName)s invoices to users who are properly registered. %(taxName)s information saved on this page will be applied to all of your account's receipts.",
+							/* translators: %s is the name of taxes in the country (eg: "VAT" or "GST") or a generic fallback string of tax names */
+							'The %(taxName)s details saved on this page will be applied to all receipts in your account.',
 							{
-								textOnly: true,
-								args: { taxName: taxName ?? translate( 'VAT', { textOnly: true } ) },
+								args: { taxName: taxName ?? fallbackTaxName },
+							}
+						) }
+						<br />
+						<br />
+						{ translate(
+							/* translators: This is a list of tax-related reasons a customer might need to contact support */
+							'If you:' +
+								'{{ul}}' +
+								/* translators: %(taxName)s is the name of taxes in the country (eg: "VAT" or "GST") or a generic fallback string of tax names */
+								'{{li}}Need to update existing %(taxName)s details{{/li}}' +
+								'{{li}}Have been charged taxes as a business subject to reverse charges{{/li}}' +
+								'{{li}}Do not see your country listed in this form{{/li}}' +
+								'{{/ul}}' +
+								'{{contactSupportLink}}Contact our Happiness Engineers{{/contactSupportLink}}. Include your %(taxName)s number and country code when you contact us.',
+							{
+								args: { taxName: taxName ?? fallbackTaxName },
+								components: {
+									ul: <ul />,
+									li: <li />,
+									contactSupportLink: (
+										<a
+											target="_blank"
+											href={ CALYPSO_CONTACT }
+											rel="noreferrer"
+											onClick={ clickSupport }
+											title={ contactSupportLinkTitle }
+										/>
+									),
+								},
+							}
+						) }
+						<br />
+						<br />
+						{ translate(
+							'For more information about taxes, {{learnMoreLink}}click here{{/learnMoreLink}}.',
+							{
+								components: {
+									learnMoreLink: (
+										<InlineSupportLink
+											supportLink={ taxSupportPageURL }
+											showText={ true }
+											showIcon={ false }
+											supportPostId={ 234670 } //This is what makes the document appear in a dialogue
```

Comment:
I'm curious where you got this from? 

IIRC `InlineSupportLink` always displays help in a dialog (but maybe I'm wrong)? [Here is one place](https://github.com/Automattic/wp-calypso/blob/21125523f7e98d17f72614cf350b3062604fb564/client/my-sites/checkout/composite-checkout/payment-methods/credit-card/assign-to-all-payment-methods.tsx#L45) that uses it in checkout. Oh, I guess that prop it overrides the `supportLink`. That's fine, but this comment is a little misleading I think?

Reply:
>I'm curious where you got this from?

Trial and error 😂 
Also, looked back to one of the PR's I worked on with Cain. 

If you don't set the post ID with the ID for the support page, the InlineSupportLink redirects to the page. If you set the `supportPostId` parameter, then it uses the dialogue.